.gitignore

```diff
@@ -52,6 +52,7 @@ package-lock.json
 !.yarn/plugins
 !.yarn/sdks
 !.yarn/versions
+install-state.gz
```

Comment:
I believe we want this check-in. Right @wespickett ?

Reply:
@Zizzamia No I don't think so, I think it should be ignored. But also shouldn't this be `.yarn/install-state.gz`? And there is already a rule (`.yarn/*`) that should ignore `.yarn/install-state.gz`

Reply:
@kyhyco let's do a solo PR for this change, so we can that merged.